ebuild-writing/misc-files/metadata/text.xml

```diff
@@ -120,7 +120,7 @@ metadata.xml:
   </ti>
 </tr>
 <tr>
-  <ti><c>&lt;stabilize-allarches&gt;</c></ti>
+  <ti><c>&lt;stabilize-allarches&gt;/</c></ti>
```

Comment:
We don't mention a closing tag for any of the other tags, so I'd rather not for reasons of consistency.

Besides, the description says "This tag must have empty content."